libs/application/core/src/lib/formUtils.ts

```diff
@@ -17,11 +15,29 @@ import {
   SubSection,
 } from '../types/Form'
 
+const containsArray = (obj: Record<string, any>) => {
```

Comment:
I see a few occurrences of `Record<string, any>` in this file. Maybe extract it to a type and have it be `Record<string, unknown>`

Reply:
I had to change a couple of places. The tests I did locally passed I will see if it passed with the CI now